### ISSUE TITLE
Thrusters rotated in flight do not update flight model

### ISSUE THREAD

**Issue by li3** [tsunamayo/Starship-EVO#2825]
### Short Description of the issue:
I have a simple ship (attached), with thrusters on a rotator, think Firefly. There's a slider to the side of the yoke that I can use to change the angle of the outrigger thrusters. When flying forward, rotating the thrusters does not change the flight model until you exit the yoke and reactivate it.

### Steps to reproduce the issue:
1. Enter the flight seat
1. Set rotator slider (right side of the yoke) to the lowest position
1. Activate the yoke
1. Fly forward until you have some velocity
1. Press tab, look at the rotator slider (right side of the yoke) and drag it to the middle position
1. Wait until ships stops, then try flying forward again.
* At this point the thrusters are pointed forward and should contribute no forward thrust anymore. However the ship will still accelerate forward.
1. Deactivate the yoke
1. Immediately re-activate the yoke
1. Try to accelerate forward, the ship will no longer accelerate

### Environment
- Starship EVO version: 20w32b
- Graphics Card: GTX 1080

### Additional information:

* Blueprint: [flight_mechanics_demo.zip](https://github.com/tsunamayo/Starship-EVO/files/5139545/flight_mechanics_demo.zip)
* Logs (though probably not useful in this case): [output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/5139547/output_log.txt)

**Comment by li3** [comment]
Note, the ship has a slider (on the left side) linked to the thrusters, and that one does control the magnitude of the thrust while in flight dynamically.

I was hoping that slider would activate the thrusters like a throttle, it doesn't appear there's any way to activate thrusters without pressing "forward" while in an active yoke.

**Comment by ZachZent** [comment]
Just so you know as this may or may not be part of the problem, but SEVO doesn't have directional thrusters in a realistic sense. Engines pointed backward contribute to thrust and lightcruise for the block engines. Brick engines in any other direction contribute to maneuverability. I'm not totally sure how engines on a child entity impact's the ship, especially if the entity maneuvers, but pointing an engine forward will not increase the breaking or create a sort of air-break.

**Comment by tsunamayo** [comment]
Yes correct, not updating thrust every frame was an optimization I did to make the game faster - but I plan to support this case. I keep you posted once it is done.
Also same for the slider, it is a valid request!
@li3 reading the case I think he understood, and actually the children should contribute to the direction they are pointing at, and they can contribute to both (but then thrust is split of course). So yes in theory you should be able to rotate your thruster to make a dropship kind of design.
Cheers
